packages/notification-center/src/i18n/languages/ca.ts

```diff
@@ -2,7 +2,7 @@ import { ITranslationEntry } from '../lang';
 
 export const CA: ITranslationEntry = {
   translations: {
-    notifications: 'Notificacions',
+    notifications: 'Notifications',
```

Comment:
Rollback this one please. The existing one is the right translation in catalonian language.

Reply:
Oops. Sorry. I should have spotted that.